huxley/core/models.py

```diff
@@ -306,7 +306,7 @@ def remove(assignment_data):
             deletions.append(assignment_data['id'])
 
         for committee, country, school, rejected in new_assignments:
-            key = (committee, country)
+            key = (committee.id, country.id)
```

Comment:
So I think the intended behavior is to have all the assignments be deleted. If you look towards the end of the method, we iterate over all assignments that weren't changed and then delete them. However, the reason why this change didn't necessarily have any effect is because when we try accessing `assigned_dict` with this key, it won't exist in it (because the keys are all IDs and not model instances). So we end up adding the assignment, and then deleting the old one later because it seems like we never updated it.